data/json/items/gun/50.json

```diff
@@ -0,0 +1,60 @@
+[
+  {
+    "id": "m107a1",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "dark_gray",
+    "name": "Barrett M107A1",
+    "name_plural": "Barrett M107A1",
+    "description": "A large, shoulder-fired, .50 caliber anti-materiel rifle.  Its large size, recoil, and noise is offset by its damage and range." ,
+    "price": 650000, "//" : "Minimum list price for the bolt-action, Gun Digest, 2012.",
+    "material": "steel",
+    "flags": "NEVER_JAMS",
+    "skill": "rifle",
+    "ammo": "50",
+    "weight": 12602,
+    "volume": 17,
+    "bashing": 12,
+    "to_hit": -1,
+    "ranged_damage": -5,
+    "range": 45,
+    "dispersion": 0,
+    "sight_dispersion": 60,
+    "aim_speed" : 6,
+    "recoil": 50,
+    "durability": 8,
+    "barrel_length" : 5,
+    "magazine_well": 1,
+    "magazines": [ [ "50", [ "m107a1mag" ] ] ],
+    "valid_mod_locations": [[ "accessories", 4 ],[ "barrel", 1 ],[ "bore", 1 ],[ "grip", 1 ],[ "mechanism", 4 ],[ "muzzle", 1 ],[ "rail", 1 ],[ "sights", 1 ],[ "stock", 1 ],[ "underbarrel", 1 ]]
+  },
+  {
+    "id": "m2browning",
+    "type": "GUN",
+    "symbol": "(",
+    "color": "dark_gray",
+    "name": "M2 Browning HMG",
+    "name_plural": "M2 Browning HMG",
+    "description": "A heavy machine gun used by the US Military from its inception to the cataclysm, and even rarely by cataclysm survivors.  Its massive size and design make it impossible to use unless deployed or mounted to a vehicle.",
+    "price": 1800000,
+    "material": "hardsteel",
+    "flags": ["MODE_BURST", "MOUNTED_GUN"],
```

Comment:
`MODE_BURST` is deprecated since the refactoring of fire modes. It's possible this should specify a single automatic fire mode if that's what would be the case IRL?